vendorconsent/tcf1/rangesection.go

```diff
@@ -154,6 +154,6 @@ type rangeVendorException struct {
 	endID   uint16
 }
 
-func (e rangeVendorException) Contains(id uint16) bool {
+func (e rangeVendorException) Contains(id uint16) bool { // TODO check if possible convert to pointer receiver
```

Comment:
I'm more skeptical of a pointer receiver here. The `rangeVendorException` is just 2 uint16 and a pointer would be at least an int32. No problem if you'd like to benchmark the difference.